deltachat-ios/Controller/AccountSetup/SecuritySettingsController.swift

```diff
@@ -1,24 +1,16 @@
 import UIKit
 import DcCore
 
+protocol SecuritySettingsDelegate: AnyObject {
+    func onSecuritySettingsChanged(newValue: Int)
```

Comment:
`options` is a list of `Int32`, what's the reason for this `Int32` -> `Int`-cast-dance? In other words: What's the reason, this parameter is of type `Int` with `options`-values being of type `Int32`?

Reply:
i added a commit that lets security-option be `Int32` throughout `SecuritySettingsController`; a cast is then needed when it comes to set/get-config then, however, that seems to be a more logical stage